coder-sdk/tags.go

```diff
@@ -66,3 +71,8 @@ func (c Client) ImageTagByID(ctx context.Context, imageID, tagID string) (*Image
 	}
 	return &tag, nil
 }
+
+// SetDefaultImageTag sets the default tag for an image. The tag must exist before calling this method.
+func (c Client) SetDefaultImageTag(ctx context.Context, imageID string, req SetDefaultTagReq) error {
+	return c.requestBody(ctx, http.MethodPatch, "/api/images/"+imageID, req, nil)
```

Comment:
This method returns the image, but I don't think that's necessary if you already have the imageID and this method returns without an error - the only changed value would be the default tag field.

Reply:
Totally agree. Returning an error alone seems appropriate here.